qa/rh/downstream.yaml

```diff
@@ -75,3 +45,10 @@ pkgs:
     - python-rbd
     - python-cephfs
     - rbd-mirror
+
+rhel_repos:
+  '7':
+    - rhel-7-server-rpms
+    - rhel-7-server-optional-rpms
```

Comment:
same https://github.com/red-hat-storage/ceph/pull/116#discussion_r461474745 , may be earlier versions ( current unsupported versions like ceph 1.x,2.x might have used optional-rpms but definitely rh-luminous dont need it as per doc)

Reply:
correct, but we install many packages based on the tests and some might be from optional-rpms. these packages are from install task.